flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "skip-icons-check": true,
+    "automerge-flathubbot-prs": true,
```

Comment:
there is no auto checking, so this is not useful at all.

Reply:
Reverted this change

Reply:
It seems the icon check skip was indeed important